chain/network/src/network_protocol/borsh.rs

```diff
@@ -11,9 +11,12 @@ use near_primitives::challenge::Challenge;
 use near_primitives::hash::CryptoHash;
 use near_primitives::network::{AnnounceAccount, PeerId};
 use near_primitives::transaction::SignedTransaction;
+use near_primitives::types::ShardId;
 use std::fmt;
 use std::fmt::Formatter;
 
+use super::StateResponseInfo;
```

Comment:
hmm, so it looks like every other `PeerMessage` variant's type that comes from `chain/network/src/network_protocol/mod.rs` is defined again explicitly here. I'm not sure what the right way to go about it is, but just pointing that out. Maybe @saketh-are knows what the best way to do it is?

related nit: There are other types imported from `super`, so this could go with them:

```diff
diff --git a/chain/network/src/network_protocol/borsh.rs b/chain/network/src/network_protocol/borsh.rs
index 621ae6bf7..3c1a47562 100644
--- a/chain/network/src/network_protocol/borsh.rs
+++ b/chain/network/src/network_protocol/borsh.rs
@@ -4,7 +4,7 @@
 //! WARNING WARNING WARNING
 //! We need to maintain backwards compatibility, all changes to this file needs to be reviews.
 use crate::network_protocol::edge::{Edge, PartialEdgeInfo};
-use crate::network_protocol::{PeerChainInfoV2, PeerInfo, RoutedMessage};
+use crate::network_protocol::{PeerChainInfoV2, PeerInfo, RoutedMessage, StateResponseInfo};
 use borsh::{BorshDeserialize, BorshSerialize};
 use near_primitives::block::{Block, BlockHeader, GenesisId};
 use near_primitives::challenge::Challenge;
@@ -15,8 +15,6 @@ use near_primitives::types::ShardId;
 use std::fmt;
 use std::fmt::Formatter;
 
-use super::StateResponseInfo;
-
 #[derive(BorshSerialize, PartialEq, Eq, Clone, Debug)]
 /// Structure representing handshake between peers.
 /// This replaces deprecated handshake `HandshakeV2`.

```